python/cudf_polars/tests/test_groupby.py

```diff
@@ -152,12 +152,17 @@ def test_groupby_len(df, keys):
     "expr",
     [
         (pl.col("int").max() + pl.col("float").min()).max(),
-        pl.when(pl.col("int") < pl.lit(2))
```

Comment:
Was this removed because it's now supported? I'm not sure I see a test case that looks similar to this one.

Reply:
Yes it's supported now see this test case https://github.com/rapidsai/cudf/pull/19242/files#diff-c0c5f4316fd6be501dbed4d33b65af567f2189868ffc54257f7493c645ae20b3R359. Although it uses, `pl.col("int") > 5`, both `5` and `pl.lit(2)` are translated as literals in the IR.

Reply:
Oh I see, yes having the `sum` inside does matter. For the second one we'd need to broadcast the `sum(float)` back to every row in the group. Hence this ternary is not supported. Thanks for pointing that out.